src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -206,6 +206,13 @@ Set<String> tabulate() throws TabulationCancelledException {
         // 2. If there's a minimum vote threshold, drop all candidates below that threshold.
         if (eliminated.isEmpty()) {
           eliminated = dropCandidatesBelowThreshold(currentRoundTallyToCandidates);
+          // One edge case: if everyone is below the threshold, we can't proceed. This would only
+          // happen in the first or (if we drop undeclared write-ins first) second round.
+          if (eliminated.size() == config.getNumDeclaredCandidates()) {
+            Logger.severe("Tabulation can't proceed because all declared candidates are below "
```

Comment:
Maybe "Tabulation complete because..." is more descriptive.

Reply:
I don't think we should consider this to be a "normal" case. Instead of producing winners and generating output files, we're aborting.